### PR TITLE
pub shuffle

### LINKED ISSUE #292: public shuffle
allow public to shuffle mint order

### PR DESCRIPTION
**Pull request by yubrew**
fix #292

please don't merge until removing the mintable tokens query

### CODE REVIEW

File: contracts/minter/src/contract_tests.rs [hunk 1]
```diff
@@ -1462,6 +1462,97 @@ fn unhappy_path() {
...
+    // setup contracts
+    let num_tokens = 20;
+    let (minter_addr, _config) = setup_minter_contract(&mut router, &creator, num_tokens);
+    // query mintable order for mints
+    let query_mintable_tokens_msg = QueryMsg::MintableTokens {};
+    let res: MintableTokensResponse = router
+        .wrap()
+        .query_wasm_smart(minter_addr.clone(), &query_mintable_tokens_msg)
+        .unwrap();
+    println!("{:?}", res);
```
Comment (penso) [inline contracts/minter/src/contract_tests.rs:1481]: Isn't it nicer to use something like `log::debug!()` instead of `println!`? Just wondering.

Comment (jhernandezb) [reply contracts/minter/src/contract_tests.rs:1481]: I have used ```dbg!()``` macro

Comment (penso) [reply contracts/minter/src/contract_tests.rs:1481]: `dbg!()` sounds like a good option too

**Comment by yubrew** [comment]
closing, part of this code was used in a different pull

**Pull request closed without merging**